test/src/main/java/org/apache/accumulo/test/functional/MemoryFreeingIterator.java

```diff
@@ -39,7 +38,12 @@ public void init(SortedKeyValueIterator<Key,Value> source, Map<String,String> op
     MemoryConsumingIterator.freeBuffers();
     while (this.isRunningLowOnMemory()) {
       // wait for LowMemoryDetector to recognize the memory is free.
-      Uninterruptibles.sleepUninterruptibly(1, TimeUnit.SECONDS);
+      try {
+        Thread.sleep(SECONDS.toMillis(1));
+      } catch (InterruptedException ex) {
+        Thread.currentThread().interrupt();
```

Comment:
I think these additions are going unchecked in the tests. `MemoryFreeingIterator` and `MemoryConsumingIterator` are used in the MemoryStarved*IT's. I don't think you can check them there as these are scan iterators. I don't think there is anything in the server-side scan code that checks this condition and relays it back to the client.

Reply:
The fact that it is never checked is likely true.  In that case, resenting it will have no effect up the stack.  Not resetting it prevents the possibility that it *will never be* handled better.  Resetting it is a better practice, so why not?